GLTFSDK/Source/SchemaValidation.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <GLTFSDK/SchemaValidation.h>
+#include <GLTFSDK/Exceptions.h>
+
+#include <memory>
+#include <sstream>
+#include <unordered_map>
+
+using namespace Microsoft::glTF;
+using namespace Microsoft::glTF::Schema;
+
+namespace
+{
+    const std::unordered_map<std::string, SchemaFlags> schemaFlagMap = {
```

Comment:
Why use this instead of just map::<std::string, std::string> like we have in SchemaJson.h?

Reply:
Because it doesn't something different? There needs to be a way to determine if a given schema uri has been flagged as disabled. How else do you suggest that happens?

Reply:
I would just disable something by setting it's schema to "{}" https://json-schema.org/